src/conways.rs

```diff
@@ -0,0 +1,263 @@
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum CellState {
+    Dead,
+    Alive,
+}
+
+#[derive(Debug)]
+pub struct Grid {
+    grid: Vec<Vec<CellState>>,
```

Comment:
can you add docs to this struct and its fields?

Reply:
Done. I added docs to conway.rs file, do you thinks is also needed in the main.rs file?